### ISSUE TITLE
Persist user groups to SSB

### ISSUE THREAD

**Issue by KyleMaas** [arj03/ssb-browser-demo#190]
So, now that we have user groups, we should figure out how best to persist this.  Any thoughts?

**Comment by arj03** [comment]
Agree, but lets get it all the features worked out so we have a better idea of what to aim for :)

**Comment by KyleMaas** [comment]
Are we comfortable enough with the way groups work to try to persist them?

**Comment by arj03** [comment]
I think so, it's still my favorite feature in browser demo :)
